docs/docsite/rst/userguide/teams.rst

```diff
@@ -97,6 +104,7 @@ Selecting the **Roles** view displays a list of the permissions that are current
 |Teams - permissions list|
 
 .. |Teams - permissions list| image:: ../common/images/teams-permissions-sample-roles.png
+   :alt: Permissions List
```

Comment:
I suggest something more descriptive along these lines:
"Permissions list with resource names, type and their associated roles."